samples/zigbee/ncp/sample.yaml

```diff
@@ -0,0 +1,21 @@
+sample:
+  name: Zigbee NCP
+  description: Zigbee network co-processor
+
+tests:
+  zigbee.ncp:
+    build_only: true
+    platform_allow: nrf52833dk_nrf52833 nrf52840dk_nrf52840 nrf52840dongle_nrf52840 nrf21540dk_nrf52840
+    tags: ci_build
+
+  zigbee.ncp.usb:
+    build_only: true
+    platform_allow: nrf52833dk_nrf52833 nrf52840dk_nrf52840 nrf52840dongle_nrf52840 nrf21540dk_nrf52840
```

Comment:
Same here: what about the `nrf21540dk_nrf52840` platform? If the FEM is soldered on the DK, the sample will not work without the FEM overlay.

Reply:
FEM-specific data is already included in device tree of `nrf21540dk_nrf52840` board. We only need overlay in case of FEM shield `nRF21540 EK`